src/unifiedreport/agent/unifiedreport.php

```diff
@@ -129,7 +129,13 @@ class UnifiedReport extends Agent
    */
   private $cpClearedGetter;
 
-  /** @var XpClearedGetter $eccClearedGetter
+
+  /** @var ipraClearedGetter $ipraClearedGetter
+   * IP clearance object
+   */
+  private $ipraClearedGetter;
+
+  /** @var eccClearedGetter $eccClearedGetter
```

Comment:
Would recommend to use the correct type hinting.

```suggestion
  /** @var XpClearedGetter $ipraClearedGetter
   * IP clearance object
   */
  private $ipraClearedGetter;

  /** @var XpClearedGetter $eccClearedGetter
```